### PR TITLE
contrib/libvirt.sh: always quietly use wget

### PR DESCRIPTION
**Pull request by MaximilianMeister**
otherwise it's too much noise logging

Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
> This was added back in - see #67.

soory haven't seen that one. the only problem is that this is really verbose, and we will store this output for many failed CI runs whhich is the bad side of it.

i think wget has some progress bar that is a bit less verbose? maybe we can trim it a bit?

**Comment by grahamhayes** [comment]
we could have a `WGET_VERBOSE` var that defaults to true, but CI sets to false?

**Comment by MaximilianMeister** [comment]
@grahamhayes thanks for the hint. updated

**Comment by grahamhayes** [comment]
I would prefer CI disable this, not have to enable it locally - the command I run is already long enough

**Comment by MaximilianMeister** [comment]
> I would prefer CI disable this, not have to enable it locally - the command I run is already long enough

that's how it works right now, no need to add anything for you

**Comment by flavio** [comment]
LGTM

File: contrib/libvirt/k8s-libvirt.sh [hunk 1]
```diff
@@ -90,10 +92,10 @@ if [ "$1" == "apply" ]; then
         # Get the SHA of the latest file
         if [ "$FLAVOUR" == "opensuse" ]; then
             echo "[+] Downloading openSUSE qcow2 VM image sha to '$IMAGE_PATH.sha256.remote'"
-            wget -q -O "$IMAGE_PATH.sha256.remote" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
+            wget $WGET_FLAGS -O "$IMAGE_PATH.sha256.remote" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
```
Comment (grahamhayes) [inline contrib/libvirt/k8s-libvirt.sh]: This can be left as a "-q" - it is only downloading a small file

File: contrib/libvirt/k8s-libvirt.sh [hunk 2]
```diff
@@ -90,10 +92,10 @@ if [ "$1" == "apply" ]; then
...
         elif [ "$FLAVOUR" == "caasp" ]; then
             echo "[+] Downloading SUSE CaaSP qcow2 VM image sha to '$IMAGE_PATH.sha256.remote'"
-            wget -q -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2.sha256"
+            wget $WGET_FLAGS -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2.sha256"
```
Comment (grahamhayes) [inline contrib/libvirt/k8s-libvirt.sh]: This can be left as a "-q" - it is only downloading a small file

File: contrib/libvirt/k8s-libvirt.sh [hunk 3]
```diff
@@ -130,12 +132,12 @@ if [ "$1" == "apply" ]; then
         if $NEED_UPDATE || $FORCE_IMAGE_REFRESH; then 
             if [ "$FLAVOUR" == "opensuse" ]; then
                 echo "[+] Downloading openSUSE qcow2 VM image to '$IMAGE_PATH'"
-                wget -O "$IMAGE_PATH" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2"
-                wget -q -O "$IMAGE_PATH.sha256" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
+                wget $WGET_FLAGS -O "$IMAGE_PATH" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2"
+                wget $WGET_FLAGS -O "$IMAGE_PATH.sha256" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
```
Comment (grahamhayes) [inline contrib/libvirt/k8s-libvirt.sh]: This can be left as a "-q" - it is only downloading a small file

File: contrib/libvirt/k8s-libvirt.sh [hunk 4]
```diff
@@ -130,12 +132,12 @@ if [ "$1" == "apply" ]; then
...
             elif [ "$FLAVOUR" == "caasp" ]; then
                 echo "[+] Downloading SUSE CaaSP qcow2 VM image to '$IMAGE_PATH'"
-                wget -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2"
-                wget -q -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2.sha256"
+                wget $WGET_FLAGS -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2"
+                wget $WGET_FLAGS -r -l1 -nd -N $CAASP_IMAGE_BASE_URL -P /tmp/CaaSP -A "SUSE-CaaS-Platform-1.0-KVM-and-Xen.x86_64*qcow2.sha256"
```
Comment (grahamhayes) [inline contrib/libvirt/k8s-libvirt.sh]: This can be left as a "-q" - it is only downloading a small file

**Comment by PI-Victor** [comment]
can you rebase so we can get this merged and closed?

**Comment by MaximilianMeister** [comment]
> This can be left as a "-q" - it is only downloading a small file

@grahamhayes I'd like to keep this consistent if you don't mind too much

**Comment by MaximilianMeister** [comment]
@mssola @PI-Victor rebased

File: contrib/libvirt/k8s-libvirt.sh [hunk 5]
```diff
@@ -137,12 +139,17 @@ if [ "$1" == "apply" ]; then
         if $NEED_UPDATE || $FORCE_IMAGE_REFRESH; then 
             if [ "$FLAVOUR" == "opensuse" ]; then
                 echo "[+] Downloading openSUSE qcow2 VM image to '$IMAGE_PATH'"
-                wget -O "$IMAGE_PATH" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2"
-                wget -q -O "$IMAGE_PATH.sha256" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
+                wget $WGET_FLAGS -O "$IMAGE_PATH" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2"
+                wget $WGET_FLAGS -O "$IMAGE_PATH.sha256" -N "http://download.opensuse.org/repositories/Virtualization:/containers:/images:/KVM:/Leap:/42.2/images/Base-openSUSE-Leap-42.2.x86_64-cloud_ext4.qcow2.sha256"
             elif [ "$FLAVOUR" == "caasp" ]; then
                 echo "[+] Downloading SUSE CaaSP qcow2 VM image to '$IMAGE_PATH'"
+<<<<<<< HEAD
```
Comment (MaximilianMeister) [inline contrib/libvirt/k8s-libvirt.sh]: oops... didn't notice there was another one

**Pull request merged**